main/level_1.gd

```diff
@@ -42,11 +44,19 @@ func _on_enemy_spawner_timer_timeout() -> void:
 		random_y += enemy_buffer_spawn_positive
 
 	new_enemy.position = Vector2(random_x, random_y)
-	add_child(new_enemy)
+	enemy_container.add_child(new_enemy)
 	
 	new_enemy.call_deferred("initialize_enemy", player)
 
 func _on_player_shoot(Bullet: PackedScene, _position: Vector2, _direction: Vector2) -> void:
 	var new_bullet = Bullet.instantiate()
 	add_child(new_bullet)
 	new_bullet.call_deferred("start", _position, _direction)
+
+func _on_waves_timer_timeout() -> void:
+	for child in enemy_container.get_children():
+		if child.has_method("queue_free"):
+			child.queue_free()
+		
+	enemy_spawner_timer.wait_time *= 0.8
+	GameManager._set_enemy_damage(GameManager._get_enemy_damage() * 1.2)
```

Comment:
Add those magic number to a variable, I think in the `GameManager` could be the right place, as `const`

Reply:
i placed it as a `var` in case we want to change that throughout the game but can change top `const` if needed